tests/pytests/test_dipoles.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+import qcelemental as qcel
+import psi4
+
+from .utils import compare_values
+
+perturbation_strength = 0.001
+
+# TODO: That "true" needs to be a string is silly. Convert it to a boolean when you can do that ithout incurring a NaN energy.
+@pytest.mark.parametrize("inp", [
+    pytest.param({'name': 'omp2', 'options': {'mp2_type': 'df', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP2'}, id='df-omp2 ae'),
+    pytest.param({'name': 'omp2', 'options': {'mp2_type': 'df', 'freeze_core': 'true', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP2'}, id='df-omp2 fc'),
+    pytest.param({'name': 'omp3', 'options': {'mp_type': 'df', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP3'}, id='df-omp3 ae'),
+    pytest.param({'name': 'omp3', 'options': {'mp_type': 'df', 'freeze_core': 'true', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP3'}, id='df-omp3 fc'),
+    pytest.param({'name': 'omp2.5', 'options': {'mp_type': 'df', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP2.5'}, id='df-omp2.5 ae'),
+    pytest.param({'name': 'omp2.5', 'options': {'mp_type': 'df', 'freeze_core': 'true', 'max_mograd_convergence': 6}, 'varname': 'DF-OMP2.5'}, id='df-omp2.5 fc'),
+    pytest.param({'name': 'olccd', 'options': {'cc_type': 'df', 'max_mograd_convergence': 6}, 'varname': 'DF-OLCCD'}, id='df-olccd ae'),
+    pytest.param({'name': 'olccd', 'options': {'cc_type': 'df', 'freeze_core': 'true', 'max_mograd_convergence': 6}, 'varname': 'DF-OLCCD'}, id='df-olccd fc')
```

Comment:
Do we need to mark this as slow?

Reply:
It timed in somewhere between 60-100 seconds when I ran it on my laptop. I can rerun it for the exact number. I want dipoles on _every_ property-producing method tested, so if it doesn't count as slow yet, it probably will before long. What's the cutoff for "slow", and how do you mark a pytest as slow?

Reply:
Updated estimate: 80 seconds at present, and hopefully we'll make the test cover more ground in future PRs.